src/components/fx/hover.js

```diff
@@ -577,9 +579,17 @@ function createHoverText(hoverData, opts) {
             ltext = label.selectAll('text').data([0]);
 
         lpath.enter().append('path')
-            .style({fill: Color.defaultLine, 'stroke-width': '1px', stroke: Color.background});
+            .style({
+                fill: commonLabelOpts.bgcolor || Color.defaultLine,
+                stroke: commonLabelOpts.bordercolor || Color.background,
```

Comment:
I've occasionally thought it would be nice to be able to see behind the hover labels - not just behind the name field like you can do now, but for example in a [fractal](http://codepen.io/alexcjohnson/full/KwQWYo/) when I want to draw a small zoom box it would be cool if the whole label were partially transparent. But in cases like that, seems like the *best* solution would be to make another attribute - `hoverlabel.opacity`, akin to `marker.opacity` so you don't need to set the background, the font color, and the border color all to rgba values - in fact, doing it that way would be a substandard experience for exactly the same reason as we have a `marker.opacity` in the first place, that you really want opacity applied to the whole group.

So yeah, I guess I'm agreeing with you that these attributes needn't support opacity on their own. If you feel like putting in `hoverlabel.opacity` now go for it, but it can also be left out of this PR and done later.